lib/Stor.pm

```diff
@@ -51,32 +116,11 @@ sub get ($self, $c) {
             payload => { statsite_key => 'error.get.malformed_sha.count' },
         }) if $sha !~ /^[A-Fa-f0-9]{64}$/;
 
-        my $tm_cache = time;
-        my $path     = $c->chi->get($sha);
-        $self->statsite->timing('cache.time', (time - $tm_cache) / 1000);
-        if ($path) {
-            $self->statsite->increment('cache.hit');
-        }
-        else {
-            $self->statsite->increment('cache.miss');
-            my $paths = $self->_lookup($sha);
-            failure::stor::filenotfound->throw(
-                {
-                    msg     => "File '$sha' not found",
-                    payload => { statsite_key => 'error.get.not_found.count' },
-                }
-            ) if !@$paths;
-
-            $path = $paths->[0];
-            $c->chi->set($sha => $path);
-        }
-
-        my $path_stat = $path->stat;
-        $c->res->headers->content_length($path_stat->size);
-        $c->res->headers->last_modified(time2str($path_stat->mtime));
 
-        $self->_stream_found_file($c, $path);
-        $self->statsite->increment('success.get.ok.count');
+        if ($ENV{TEST} || !$self->get_from_hcp($c, $sha)) {
```

Comment:
This change broke current Stor instances... S3 backend should be opt-in by configuration...